src/devTools.js

```diff
@@ -247,11 +250,14 @@ export const ActionCreators = {
   setMonitorState(monitorState) {
     return { type: ActionTypes.SET_MONITOR_STATE, monitorState };
   },
-  recomputeStates(committedState, stagedActions) {
+  recomputeStates(committedState, stagedActions, timestamps, currentStateIndex=-1, skippedActions={}) {
```

Comment:
Probably this should be refactored to recomputeStates(devState) or better import(devState), getting rid of all these parameters and in that way making sure, that future additions to the devState will create an api change.